src/System.Private.CoreLib/src/System/String.CoreCLR.cs

```diff
@@ -29,7 +29,7 @@ public partial class String
         // We need to call the String constructor so that the compiler doesn't mark this as a literal.
         // Marking this as a literal would mean that it doesn't show up as a field which we can access 
         // from native.
-        public static readonly string Empty;
+        [Intrinsic] public static readonly string Empty;
```

Comment:
Nit: The common style in coreclr/corefx is place the field attribute on its own line.

Reply:
I know, I was just going with the style a couple lines above. I can fix those to the common style, but I'm not a huge fan of reformatting things because it makes digging through history harder.

Reply:
I see - you have tried to match the rest of the file. I would fix the style in the rest of the file instead.